.github/workflows/release.yml

```
@@ -1,13 +1,26 @@
 name: Release
 
 on:
-  schedule:
+  #schedule:
   # Every Monday at 07:00 (UTC)
-  - cron: "00 7 * * MON"
+  #- cron: "00 7 * * MON"
   workflow_dispatch:
     inputs:
+      release_type:
+        description: 'Release type'     
+        required: true
+        type: choice
+        options:
+        - major
+        - minor
+        - patch
+      release_branch:
+        description: Branch to release
+        required: true
+        default: master
+        type: string
       quay_repository:
-        description: Quay repository
+        description: Quay repository    
```

trivial but it caught my eye - end of line spaces irk me :)
```suggestion
        description: Quay repository
```